scripts/kubernetes/manifests/monitoring/prometheus-monitor/rules.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: prometheus-rules
+  labels:
+    name: prometheus-rules
+  namespace: bassa-monitoring
+data:
+  alert.rules: |-
+    groups:
+      - name: Deployment
+        rules:
+        - alert: Deployment at 0 Replicas
+          annotations:
+            summary: Deployment {{$labels.deployment}} in {{$labels.namespace}} is currently having no pods running
+          expr: |
+            sum(kube_deployment_status_replicas{pod_template_hash=""}) by (deployment,namespace)  < 1
+          for: 1m
+          labels:
+            name: 'bassa-mail-notifications'
+      - name: Pods
+        rules:
+        - alert: Container restarted
+          annotations:
+            summary: Container named {{$labels.container}} in {{$labels.pod}} in {{$labels.namespace}} was restarted
+          expr: |
```

Comment:
Is it possible to add some documentation about this expressions?

Reply:
Yeah sure 😁

Reply:
@JaDogg Added the comment and [wiki](https://github.com/scorelab/Bassa/wiki/Prometheus-for-monitoring-and-alerting-K8s-cluster) as well :)